addons/project/static/src/project_sharing/views/form/project_sharing_form_compiler.js

```diff
@@ -65,7 +65,7 @@ export class ProjectSharingChatterCompiler extends ViewCompiler {
         return compileChatter(node, {
             resId: 'model.root.resId or undefined',
             resModel: 'model.root.resModel',
-            projectSharingId: 'model.root.context.active_id',
+            projectSharingId: 'model.root.context.active_id_chatter',
```

Comment:
```suggestion
            projectSharingId: 'model.root.context.default_project_id',
```
:angel:

Reply:
nvm, it is popped from the context when arriving in the form view.

Reply:
I am not sure to see why the `default_project_id` is poped in the form view. 🤔

Reply:
indeed it is popped as discussed with @auon-odoo 🙂